components/Builder.vue

```diff
@@ -97,13 +107,60 @@
           :features="component.features.data"
         />
       </div>
+
+      <div
+        class="background-gradient-lines"
+        v-else-if="typeOf(component) == 'clubs'"
+      >
+        <Clubs
+          :clubs="component.clubs.clubs"
+          :color="component.clubs.color"
+        />
+      </div>
+
+      <div
+        class="background-gradient-lines"
+        v-else-if="typeOf(component) == 'description'"
+      >
+        <Description
+          :quotes="component.description.quotes"
+        />
+      </div>
+
+      <div
+        class="background-gradient-lines"
+        v-else-if="typeOf(component) == 'social'"
+      >
+        <Social
+          :facebook="component.social.facebook"
+          :instagram="component.social.instagram"
+          :snapchat="component.social.snapchat"
+          :tiktok="component.social.tiktok"
+          :mail="component.social.mail"
```

Comment:
This mistake is present in #59 (see https://github.com/BDE-Polytech-MTP/website-front/pull/59#discussion_r705022128 and https://github.com/BDE-Polytech-MTP/website-front/pull/59#discussion_r705021535) as well.

**I will not be pointing out any errors that were reported previously and will only discuss it in the first PR.**